content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -1,72 +1,240 @@
-====================================================================
-When Should you Use Packages, Units of Measure or Special Packaging?
-====================================================================
+==========================================
+Units of measure, packages, and packagings
+==========================================
 
-Units of Measure
+In Odoo, there are a variety of ways to specify the amount of products being bought, stocked, and
+sold. *Units of measure*, *packages*, and *packagings* are all available to streamline inventory
+flows, allowing for a variety of configurations for products as they enter and leave the warehouse.
+
+Units of measure
 ================
 
-Units of measure specify the unit used to handle a product. In Odoo, you
-have the possibility to specify the unit of measure in which you manage
-your stock and the one which is used when purchasing the product to your
-supplier.
+*Units of measure (UoM)* refers to different measurable quantities used to handle products, such as
+units, weight, time, or size. Different :abbr:`UoMs (Units of Measure)` for weight can be kilos,
+pounds, ounces, grams, etc.
+
+In Odoo, stock management and purchasing from suppliers are streamlined by specifying different
+:abbr:`UoM (Units of Measure)` for both buying and selling products.
+
+.. image:: usage/uom-on-product-form.png
+   :align: center
+   :alt: Specify unit of measure for selling a product vs purchasing.
+
+Once a product has a default :guilabel:`Unit of Measure` and :guilabel:`Purchase Unit of Measure`
+set on the product form, Odoo automatically converts the different units in the product's
+purchase/sales orders and the corresponding delivery orders/receipts.
+
+The only condition is that all of the units have to be in the *same category* (unit, weight, volume,
+length, etc.).
+
+.. example::
+   On the product form for `Rope`, the following fields are set as:
 
-.. image:: usage/usage_01.png
-    :align: center
+   - :guilabel:`Unit of Measure` in `ft` (feet), and
+   - :guilabel:`Purchase Unit of Measure` in `cm` (centimeters).
 
-The *conversion* between the different units of measure is done
-automatically. The only condition is that all the units have to be in
-the *same category* (Unit, Weight, Volume, Length,...)
+   Because the vendor sells rope in `cm`, the purchase :guilabel:`UoM ` is used to represent the
+   quantity on the :abbr:`PO (Purchase Order)`.
```

Comment:
If a reader can't load the screenshot here, then this sentence is difficult to interpret.
I therefore think it's worth spelling this out, since images are only meant to be supplemental.